src/components/UserData.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+UserData.propTypes = {
+  personalUserData: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      phone: PropTypes.string.isRequired,
+      mail: PropTypes.string.isRequired,
+      department: PropTypes.string.isRequired,
+    })
+  ),
+  skills: PropTypes.arrayOf(
+    PropTypes.shape({
+      skill1: PropTypes.string.isRequired,
+      skill2: PropTypes.string,
+      skill3: PropTypes.string,
+      skill4: PropTypes.string,
+      skill5: PropTypes.string,
+      skill6: PropTypes.string,
+    })
+  ),
+}
+
+export default function UserData({ personalUserData, skills }) {
```

Comment:
This name is a bit misleading. When I read it in another file, I thought it'd data coming from a json file. How about just `User` or `UserCard` or something like that?